src/Simplify.Web/Model/Validation/Attributes/MinAttribute.cs

```diff
@@ -36,22 +87,32 @@ public override void Validate(object? value, PropertyInfo propertyInfo, IDIResol
 	{
 		if (value == null)
 			return;
-		
-		if (value is not IComparable comparableValue)
-			throw new ArgumentException($"The type of specified property value must be inherited from {typeof(IComparable)}");
 
-		ValidateTypesMatching(comparableValue);
-		
+		var minValue = ConvertToIComparable(MinValue);
+		var comparableValue = (IComparable)value;
```

Comment:
if `value` is not `IComparable` you will get cast exception. Need to check it for `IComparable` and add a test for that case.